src/components/victory-area/victory-area.js

```diff
@@ -139,12 +139,16 @@ class VictoryArea extends React.Component {
     );
   }
 
+  shouldAnimate() {
+    return Boolean(this.props.animate);
```

Comment:
It's not a huge issue, but using `!!` instead of `Boolean` would be *slightly* safer, since `Boolean` can be overloaded.